ethereum_history_api/oracles/src/script/noir_fixtures/boundedVec.ts

```diff
@@ -1,6 +1,9 @@
 import { Hex } from 'viem';
 import { indentBlock, joinArray, joinArrayVertical } from '../../noir/noir_js/encode.js';
 import { encodeHex } from '../../noir/oracles/common/encode.js';
+import { ZERO_PAD_VALUE } from '../../noir/oracles/common/const.js';
+import { padArray } from '../../util/array.js';
+import { BYTE_HEX_LEN } from '../../util/const.js';
 
 export function createBoundedVecFixture(value: Hex): string {
```

Comment:
We now have two functions:
`createBoundedVecFixture`
and
`createBoundedVecFixtureWithLen`

The second one is the default as values in `BoundedVec` are usually padded. The first case is kinda an exception.
So let's call them `createFocusedBoundedVecFixture` & `createBoundedVecFixture`